pennylane/ops/functions/equal.py

```diff
@@ -56,38 +56,33 @@ def equal(
     rtol=1e-5,
     atol=1e-9,
 ) -> bool:
-    r"""Function for determining operator or measurement equality.
+    r"""Function for determining operator, measurement, and tape equality.
 
     .. Warning::
 
-        The ``qml.equal`` function is based on a comparison of the type and attributes
-        of the measurement or operator, not a mathematical representation. While
-        comparisons between some classes, such as ``Tensor`` and ``Hamiltonian``, are
-        supported, mathematically equivalent operators defined via different classes
-        may return False when compared via ``qml.equal``.
-
-        To be more thorough would require the matrix forms to be calculated, which may
-        drastically increase runtime.
+        The ``qml.equal`` function is based on a comparison of the types and attributes of the
```

Comment:
How is this block changed? Hard to notice the difference.

Reply:
Mostly some minor changes in wording:

type and attributes -> types and attributes
measurement or operator -> measurements or operators
not a mathematical representation -> not their mathematical representations
while comparisons between some classes, ..., are supported -> while mathematical comparisons ...